### PR TITLE
Add multi color support to continuous color legend

### PR DESCRIPTION
**Pull request by matthewma7**
image

### CODE REVIEW

File: tests/cases/colorLegend.js [hunk 1]
```diff
@@ -67,6 +67,13 @@ describe('color legend', function () {
       base: Math.E,
       domain: [100, 10000],
       colors: ['blue', 'olive']
+    },
+    {
+      name: 'Continuous multicolor',
+      type: 'continuous',
+      scale: 'pow',
+      domain: [100, 1000],
```
[the remark this answers is not in the document]
Comment (matthewma7) [reply tests/cases/colorLegend.js]: The current code doesn't support piecewise function, the workaround is creating multiple color ranges. Do we need piecewise support?

[the remark this answers is not in the document]
Comment (aashish24) [reply tests/cases/colorLegend.js]: @matthewma7 I think we want piecewise support in GeoJS. I am assuming that it just request the JavaScript code, the SVG can use the result of the piecewise function.

[the remark this answers is not in the document]
Comment (matthewma7) [reply tests/cases/colorLegend.js]: I see, I'll add piecewise function support to this PR.

**Comment by aashish24** [comment]
@kkola

**Pull request merged**